Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -1149,15 +1146,21 @@ def fetch_incidents(client: Client,
 def main():
     os.environ['PAN_CREDENTIALS_DBFILE'] = os.path.join(gettempdir(), 'pancloud_credentials.json')
     params = demisto.params()
-    registration_id_and_url = params.get(REGISTRATION_ID_CONST).split('@')
+    registration_id_and_url = (params.get('credentials_id', {}).get('identifier') or params.get('reg_id')).split('@')
```

Comment:
Where is this in the yml?
I think you missed it and it should be part of credentials_auth_key (as per your yml)

Reply:
I renamed it now so it would be less confusing